readline.go

```diff
@@ -304,6 +302,11 @@ func (i *Instance) EnableHistory() {
 	i.operation.history.Enable()
 }
 
+// ClearScreen clears the screen.
+func (i *Instance) ClearScreen() {
```

Comment:
I'm not very familiar with the internals of this library, but in my experiments I had to call `o.Clean()` and `o.Refresh()` as well. Should this be part of the `ClearScreen` method?

Reply:
If you write to `(*Instance).Stdout()` or similar, it will refresh automatically via `wrapWriter`:

https://github.com/ergochat/readline/blob/eb5f59c8e9aef537572f8828f0cf169ce0beddc2/operation.go#L41